lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -1,5 +1,6 @@
-define (require, exports, module) ->
-  EQUATION_STATE = require('./papeeria_latex_highlight_rules').EQUATION_STATE
+define( (require, exports, module) ->
+  papeeria_latex_highlight_rules = require('./papeeria_latex_highlight_rules')
```

Comment:
unless I don't see something between the lines, the number of `require`calls didn't change.